Master-Game/Assets/Maps/TreasureHunterTimBasicMap.json

```diff
@@ -0,0 +1,91 @@
+﻿{
+  "name":"Basic Treasure Hunter map",
+  "version":"1.0.0",
```

Comment:
let's be consistent on versioning. @jamiahuswalton  has 1.0 and this file has 1.0.0 I'll let you guys decide which way you want to go with it.

Reply:
I am not sure what the normal convention is for versions. What do the numbers mean?

Reply:
Major version, Minor version, Revision.
So for a map if the map was radically changed then we change the major version.
If map goes through some iteration/design changes then it would be a minor version bump
if the map is tweaked then Revision.

But I think we can handle that in two digits also. Major and Minor.